My recent work branch

Marlyn-Mayienga opened the pull request:
# # My recent_work and contact_form branch:
-created the recent_work branch
-modified the HTML file to create the recent work section
-modified the HTML file to create the contact form section
-created the contact form section
-modified the contact form on CSS
-added an image to the recent work section
-added a footer with links to my social pages

Marlyn-Mayienga committed:
removed the <br> tags on the paragraph

Marlyn-Mayienga committed:
made the following changes:
created the recent works branch
modified the html recent works section
created the contact form section
modified the contact form on css
added an image to the recent work section
added a footer

mikethreels reviewed (changes requested):
Hi Marlyn, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is not passing all the linter checks, please make sure that all linters are passing before submitting your code for review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

Marlyn-Mayienga committed:
fix the linter errors in the html file

Marlyn-Mayienga committed:
fix the linter errors in style.css

Marlyn-Mayienga committed:
fixed linter errors in the html file

Marlyn-Mayienga committed:
fix the html linter errors

mikethreels commented on index.html:
```diff
@@ -1,108 +1,280 @@
+          <div class="work-item">
+            <img class="work-img" src="./project.jpg" alt="project image"/>
+            <!-- <img class="shadow-img" src="./Ellipse 19.svg" alt="project image"/> -->
+            <img class="shadow-img" src="./Ellipse.svg" alt="project image"/>
```
- [ ] decorations should always be backgrounded images, not `<img>` tags

mikethreels commented on index.html:
```diff
@@ -1,108 +1,280 @@
-                </div>
+      <div class="divider"></div>
+      <!-- TECHNLOGIES -->
+      <section class="technologies">
+        <div class="technologies-item">
+          <h2 class="technologies-title">Languages</h2>
+          <ul>
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+        <div>
+          <div class="technologies-item">
+            <div class="technologies-title">Frameworks</div>
+            <ul>
+              <li>BootStrap</li>
+              <li>Ruby on Rails</li>
+              <li>RPSec</li>
+              <li>CapyBase</li>
+              <li>Selenium</li>
+            </ul>
+          </div>
+        </div>
+        <div>
+          <div class="technologies-item">
+            <div class="technologies-title">Skills</div>
+            <ul>
+              <li>Codekit</li>
```
- [ ] every separate list has a header and all headers should have the same styling as you can see in the below print screen this is not the case
![Screenshot 2022-04-24 000729](https://user-images.githubusercontent.com/39240820/164947483-0e9525b4-4c67-4db6-8117-0735cb72fe79.png)
.

mikethreels commented on index.html:
```diff
@@ -1,108 +1,280 @@
-            and experience!If <br>
-            you like what you see and have a project you need<br> coded, don’t hestiate to contact me.</p>
+      <h1 class="name">Marlyn Mayienga</h1>
+      <img class="first-img" src="./circle_header.svg" alt="circle 2 header"/>
```
- [ ] decorations should always be backgrounded images, not `<img>` tags

mikethreels commented on index.html:
```diff
@@ -1,108 +1,280 @@
+          </div>
+        </div>
+      </section>
+      <img class="second-img" src="./circle_header.svg" alt="circle 2 header"/>
```
- [ ] decorations should always be backgrounded images, not `<img>` tags

mikethreels commented on index.html:
```diff
@@ -1,108 +1,280 @@
+        <div>
+          <p>Work &darr;</p>
+        </div>
+        <div class="social-links">
+          <a href="https://github.com/Marlyn_Mayienga"
+            ><i class="fab fa-github"></i
+          ></a>
+          <a href="https://www.linkedin.com/in/marlyn-mayienga/"
+            ><i class="fab fa-linkedin"></i
+          ></a>
+          <a href="https://www.instagram.com/merl.__/?hl=en"
+            ><i class="fas fa-cat"></i
+          ></a>
+          <a href="https://twitter.com/M_ayienga"
+            ><i class="fab fa-twitter"></i
+          ></a>
+          <a href="marlymayienga@gmail.com"><i class="fas fa-envelope"></i></a>
```
- [ ] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)

mikethreels reviewed (changes requested):
Hi Marlyn, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Marlyn-Mayienga committed:
made the following changes:
changed the social and footer links tag to ul tags
corrected the error on the languages section
made the images background images
created a modal for the first project sample

phelian23 commented on style.css:
```diff
@@ -180,7 +180,293 @@ body {
+  padding: 20px 20px;
+}
+
+@media only screen and (max-width: 770px) {
+  .nav-bar {
+    display: none;
+  }
+
+  .hamburger-btn {
+    color: white;
+    display: block;
+    font-size: 2em;
+  }
+
+  .technologies {
+    justify-content: flex-start;
+  }
+
+  .page-break {
+    flex-direction: column;
+    align-items: center;
+  }
+
```
- [ ] In the mobile version, kindly remove the horizontal scroll present.
![Capture](https://user-images.githubusercontent.com/85095702/165326092-e8f09d51-d53d-4304-8eee-5e6a890b97a5.PNG)

I think it's the background image causing the width to overflow

Marlyn-Mayienga replied:
@phelian23 
Could you help me identify the exact place that is causing the overflow please if you don't mind? I don't seem to see the error from my end.

phelian23 reviewed (changes requested):
Hi @Marlyn-Mayienga, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Linters are passing
✅ Descriptive pull request
✅ Good commit messages
✅ Good UI design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Marlyn-Mayienga committed:
fix the second image to a background-image

Marlyn-Mayienga committed:
fixed linter errors

Marlyn-Mayienga committed:
fix the divider to hr tags and changed the colour

Marlyn-Mayienga committed:
fix the linter errors

ShahierNashaat reviewed (approved):
Hi @Marlyn-Mayienga , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Marlyn-Mayienga committed:
remove all the commnted code to make work look neat
